packages/pdfshift/configuration-schema.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "properties": {
+    "baseUrl": {
+      "title": "Base URL",
+       "type": "string",
+      "description": "The base PDFShift URL",
+      "format": "uri",
+      "minLength": 1,
+      "examples": ["https://api.pdfshift.io"]
+    },
+    "apiVersion": {
+      "title": "API Version",
+      "anyOf": [
+        {
+          "type": "string"
+        },
+        {
+          "type": "null"
+        }
+      ],
+      "placeholder": "v3",
+      "description": "PDFShift api version",
+      "minLength": 1,
+      "examples": [
+        "v3"
+      ]
+    },
+    "apiKey": {
+      "title": "API Key",
+      "type": "string",
+      "description": "Your PDFShift API key",
+      "writeOnly": true,
+      "examples": ["your-pdfshift-api-key"]
+    }
+  },
+  "type": "object",
+  "additionalProperties": true,
+  "required": ["baseUrl", "apiKey"]
```

Comment:
If we default `baseUrl: "https://api.pdfshift.io"` we should remove it from the required list